src/fides/api/ops/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -69,3 +97,119 @@ def privacy_experience_list(
             results.append(privacy_experience)
 
     return fastapi_paginate(results, params=params)
+
+
+@router.post(
+    urls.PRIVACY_EXPERIENCE,
+    status_code=HTTP_201_CREATED,
+    response_model=List[PrivacyExperienceResponse],
+    dependencies=[
+        Security(verify_oauth_client, scopes=[scope_registry.PRIVACY_EXPERIENCE_CREATE])
+    ],
+)
+def privacy_experience_create(
+    *,
+    db: Session = Depends(deps.get_db),
+    bulk_experience_data: conlist(PrivacyExperienceCreate, max_items=50),  # type: ignore
+) -> List[PrivacyExperience]:
+    """
+    Bulk create Privacy Experiences
+    """
+    logger.info("Creating privacy experiences")
+    experiences: List[PrivacyExperience] = []
+    for experience_data in bulk_experience_data:
+        experience = PrivacyExperience.create(
+            db, data=experience_data.dict(exclude_unset=True), check_name=False
+        )
+        # Temporarily stash the privacy notices on the experience for display
+        experience.privacy_notices = experience.get_related_privacy_notices(db)
```

Comment:
oh is the idea that during one session, each `experience` will have `privacy_notices` hanging around?

Reply:
Yes this is just temporarily saving these privacy notices on the experience in memory.  A separate request to get experiences would recalculate this.